### PR TITLE
feature(kensetsu): remove managers

### LINKED ISSUE #1029: [Tech]: Permissions on vault parameters
### Problem statement

We want the management of vault parameters to be managed by the community. Not risk manager assigned with particular rights on managing the vault parameters - for legal reasons. 

All update of any vault parameter should go through governance as any usual vote.

### Description

_No response_

### Definition of Done

- [ ] All vault parameter update goes through Root permission.

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by Alexey-N-Chernyshov**
Replaces managers with governance.
Resolves #1029

### CODE REVIEW

File: pallets/kensetsu/src/lib.rs [hunk 1]
```diff
@@ -773,13 +757,16 @@ pub mod pallet {
         /// - `kusd_amount`: The amount of stablecoin (KUSD) to withdraw as protocol profit.
```
Comment (wer1st) [inline pallets/kensetsu/src/lib.rs:757]: add `beneficiary`

File: pallets/kensetsu/src/lib.rs [hunk 2]
```diff
@@ -470,8 +460,6 @@ pub mod pallet {
         CDPUnsafe,
         /// Too many CDPs per user
         CDPLimitPerUser,
-        /// Risk management team size exceeded
-        TooManyManagers,
```
Comment (wer1st) [inline pallets/kensetsu/src/lib.rs:474]: Do we already have Kensetsu on prod?
If yes, the shifting of error id could corrupt the history of indexers.

@vovac12 what do you think? Can we just remove it or leave as "unused error"?

**Review by wer1st — COMMENTED** [review]
Should we update `STORAGE_VERSION` in the pallet?

File: pallets/kensetsu/src/lib.rs (hunk 2 above)
Comment (Alexey-N-Chernyshov) [reply pallets/kensetsu/src/lib.rs:474]: No, we do not have Kensetsu on prod

**Review by wer1st — APPROVED** [review]

**Review by vovac12 — APPROVED** [review]

**Pull request merged**